velox/functions/prestosql/aggregates/tests/ChecksumAggregateTest.cpp

```diff
@@ -162,6 +162,30 @@ TEST_F(ChecksumAggregateTest, varchars) {
       {"Thanks \u0020\u007F"_sv}, "oEh7YyEV+dM=");
 }
 
+TEST_F(ChecksumAggregateTest, shortDecimal) {
+  assertSingleGroupChecksum<int64_t>({{}}, "h8rrhbF5N54=", DECIMAL(10, 5));
```

Comment:
Add a test for the max and min decimal values.

Reply:
Good suggestion, I'll add it